hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1543,7 +1593,7 @@
       name: chip_csrng_lc_hw_debug_en
       desc: '''Verify the effect of LC HW debug enable on CSRNG.
 
-            TODO: This is pending SCA security review and might be removed.
+            TBD.
```

Comment:
the effect of LC_HW now is only to select which diversification hardware vector should be mixed in with the incoming seed from ES.

Reply:
I updated this test to do a read of `int_state_val` CSR instead.